src/state.cpp

```diff
@@ -905,31 +986,35 @@ State::apply(const std::vector<CachedProposal>& proposals)
 /// Message protection
 ///
 
-MLSCiphertext
-State::protect(const bytes& pt)
+MLSMessage
+State::protect_app(const bytes& pt, const MessageOpts& msg_opts)
```

Comment:
Change signature to take `(authenticated_data, padding_size)` instead of `MessageOpts`, since `encrypt` isn't relevant.